src/azure-cli/azure/cli/command_modules/acr/_params.py

```diff
@@ -436,6 +436,9 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('remove_repos', options_list=['--remove'], nargs='*', required=False,
                    help='respsitory permissions to be removed from the targeted connected registry and it\'s succesors sync scope maps. Use the format "--remove [REPO1 REPO2 ...]" per flag. ' + repo_valid_actions)
 
+    with self.argument_context('acr connected-registry install') as c:
+        c.argument('parent_protocol', options_list=['--parent-protocol'], help='Required parameter to specify the parent protocol. Should be either "http" or "https".', required=True)
```

Comment:
Can the argument be defined as an enum type? Something like https://github.com/Azure/azure-cli/blob/29767d75d850ddc1c24cc85bd46d861b61d77a47/doc/authoring_command_modules/authoring_commands.md#registering-enums

Reply:
@rosanch @huanwu @jaysterp

Reply:
Changed